styles/folders.less

```diff
@@ -124,6 +124,13 @@
     padding: var(--input-text-ph) var(--input-text-pw);
 
     &.disabled { color: var(--disabled-fg); }
+    &:not(.disabled) {
+        &.has-tabs {
+            color: var(--userlink-fg);
+            filter: saturate(0.7);
+            font-style: italic;
```

Comment:
Italics are typically used for status/flavor text like the "What's New" link at the very bottom of the UI, so italicizing the title seems out of place in this context.

(And re. my point about the CSS class above, here you can select it with `.has-open-tabs &:not(.disabled)`.)

Reply:
Italic style removed and I refactored the class as you suggested.